homework/index.js

```diff
@@ -29,22 +29,97 @@
     return elem;
   }
 
+  function createTableRows(table, optionsHeader = {}, optionsValue = {}) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('td', tr, optionsHeader);
+    if (optionsHeader.text === 'Repository:') {
+      const td = createAndAppend('td', tr);
+      createAndAppend('a', td, optionsValue);
+    } else {
+      createAndAppend('td', tr, optionsValue);
+    }
+  }
+
+  function formatDate(dateString) {
+    const date = new Date(dateString);
+    let hours = date.getHours();
+    const ampm = hours >= 12 ? 'PM' : 'AM';
+    hours %= 12;
+    hours = hours === 0 ? 12 : hours;
+    let seconds = date.getSeconds();
+    seconds = seconds < 10 ? `0${seconds}` : seconds;
+    const formatedDate = `${date.getMonth() +
+      1}/${date.getDate()}/${date.getFullYear()}, ${hours}:${date.getMinutes()}:${seconds} ${ampm}`;
+    return formatedDate;
+  }
```

Comment:
There is no need to do this formatting yourself. The `Date` object has many built-in ways to format a date. Another student posed a question about this in slack:

https://hackyourfuture.slack.com/archives/CL795TNFQ/p1568802816003000

I would just use `.localLocaleString()`. This uses the date format that the user has chosen for his/her computer.

Reply:
Very efficient indeed! I can't believe I missed this.. Thank you very much.